docs/docs/services/qix-engine/access-control.md

```diff
@@ -1,76 +1,132 @@
+Being able to control which users can do what in the context of a document is often necessary. In Qlik Associative
+Engine we support [Attribute-Based Access Control (ABAC)](https://en.wikipedia.org/wiki/Attribute-based_access_control)
+which lets you control the document resources based on users' attributes rather than using roles or other more static
+means.
 
-## Configuration
+## Rules
 
-Rules are defined, one on each row, in text files. Qlik Associative Engine will
-apply rules in order (except for deny rules which are always applied first,
-regardless of where they are defined).
+Each rule is a conditional expression that evaluates to either `true` or `false`. In addition to this, each rule
+contains a mandatory part that determines which actions the rule grants, or denies.
 
-If the ABAC feature is enabled, all users will by default have no
-access, so it is important that you also include rules when enabling it
-or you will cause access problems for your users.
+!!! Note
+    If the ABAC feature is enabled, all users will by default have no access, so it is important that you also include
+    rules when enabling it or you will cause access problems for your users.
+
+### Rules Overview
+
+Upcoming sections more formally describe how rules are formed, but to get an initial understanding, consider the
+following rule:
+
+```py
+user.id = "ada-lovelace" and resource._resourcetype = "App" and resource._actions = {"create", "update", "read"}
+```
+
+This rule states that if the user is `ada-lovelace` and the accessed resource type is an `App`, grant the actions
+`create`, `update`, and `read`.
+
+The first two comparisons in the expression are optional. They could be omitted and the expression would still be a
+valid rule. In this example they must evaluate to `true` in order for the actions to be granted.
+
+The expression involving `_actions` must be present in order to determine which actions that are granted access. See
+section [Actions](#actions) for more details.
+
+## Rule Files
+
+Rules are defined in text files with one rule on each row. Qlik Associative Engine applies rules in the order they
+appear in the file. Two types of rule files are supported. The _Deny_ rule file and the _Allow_ rule file.
+
+The Deny rule file denies access and is always evaluated first. If any rule in the Deny file evaluates to `true`,
+access is immediately denied for the actions proviced and no further evaluation of rules take place. 
+
+The Allow rule file allows access. If a rule in the Allow file evaluates to `true`, access is granted for the
+actions specified and rule evaluation continues, possibly accumulating access to more actions based on succeeding
+rules.
+
+## Engine Configuration for ABAC
+
+### ABAC-related Command Line Switches
+
+| Switch | Values | Description |
+| ------ | ------ | ----------- |
+| `EnableABAC` | `0` or `1` | Enabling/disabling of ABAC rule evalutation. |
```

Comment:
Do we have anywhere else in the Qlik Core docs where these should be added instead? And then just refer to them here?

Reply:
Agree, but I won't do that as part of this task.